src/org/javarosa/core/io/StreamsUtil.java

```diff
@@ -4,8 +4,11 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.OutputStream;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class StreamsUtil {
+    private static final Logger logger = LoggerFactory.getLogger(StreamsUtil.class);
```

Comment:
Is this used?

Reply:
I see two `StreamsUtil` classes on `org.javarosa.core.util` and `org.javarosa.core.io` and none are used by JR. It's alarming how they're basically the same classes with slightly different implementations for handling streams. 

I think this classes can be safely removed:
- JR itself doesn't use them.
- No one should depend on a JR's implementation of stream handling functions. There are better and far more battle-tested alternatives like commons-io and anyone relying on these classes should migrate their code a.s.a.p.

Reply:
👍 How about marking both as deprecated?

Reply:
I'm still concerned... I think we should break build if we detect deprecation warnings.